frontend/src/components/common/header/index.css

```diff
@@ -150,3 +150,40 @@ a.dropdown-item {
     display: none;
   }
 }
+
+.resource-heading {
+  margin-top: 60px;
+  margin-bottom: 60px;
+  font-size: 48px;
+}
+
+.resource-title {
+  margin-top: 60px;
+  margin-top: 60px;
+}
+
+.resource-p {
```

Comment:
lets try and come up with a better class name here something a bit more descriptive then p and p2

maybe get rid of resource-p and just do it on the `p` and then instead of `p2`, maybe a class name of `subparagraph`

so 

```
p {
  margin-bottom: 60px;
  font-size: 24px
}

.subparagraph {
  margin-bottom: 0
  font-size: 20px
}
```